examples/server/index.html

```diff
@@ -30,15 +30,15 @@
 
 <h2>Options</h2>
 <div class="option">
-    <input id="use-datachannel" checked="checked" type="checkbox"/>
+    <input id="use-datachannel" type="checkbox"/>
     <label for="use-datachannel">Use datachannel</label>
 </div>
 <div class="option">
-    <input id="use-audio" checked="checked" type="checkbox"/>
+    <input id="use-audio" type="checkbox"/>
```

Comment:
Same as above

Reply:
Sorry about that. I changed to test faster and I forgot to revert before committing